scan_form.go

```diff
@@ -36,6 +36,7 @@ type ListScanFormsResult struct {
 	HasMore bool `json:"has_more,omitempty"`
 }
 
+// TODO: Fix function name
```

Comment:
In golang, lowercased function names are not public meaning you should be able to correct this without affecting users.